bokeh/embed.py

```diff
@@ -40,19 +40,41 @@ def components(plot_object, resources=None):
               are **already loaded**.
 
     Args:
-        plot_object (PlotObject) : Bokeh object to render
-            typically a Plot or PlotContext
+        plot_objects (PlotObject|list|dict|tuple) : Will return script
+        and div element to render if PlotObject, or correlating structures
+        of script/div combinations
         resources : Deprecated argument
     Returns:
-        (script, div) : UTF-8 encoded
-
+        (script, div): UTF-8 encoded|list|dict|tuple
```

Comment:
This is also sophisticated enough that it probably warrants a `Notes` or `Examples` section that shows the correlated inputs/outputs. Something like:
```
Notes: 
     
    .. code-block: python
        components(foo)                      # (script, foo_div)

        components((foo, bar))               # (script, (foo_div, bar_div))

        components({"foo": foo, "bar": bar}) # (script, {"foo": foo_div, "bar": bar_div})